### DISCUSSION TITLE
Any plan to make a windows 11 "skin" ?

### DISCUSSION THREAD

**Discussion by floraaubry** [xanderfrangos/twinkle-tray#342] [Ideas]
Hello, as the title says, i wanted to know if you plan to do a windows 11 like ui for Twinkle tray !
Love your app, helps me alot, thank you :)

**Comment by clothedsnake** [comment]
It's in the latest beta: https://github.com/xanderfrangos/twinkle-tray/issues/296

**Comment by sapphired18** [comment]
and windows 11 22H2 brightness slider too!'
ex. 
![image](https://user-images.githubusercontent.com/58063311/177039904-983b4ebc-f390-4a43-8310-e53d177388ac.png)
